doc/rbd/rbd-persistent-cache.rst

```diff
@@ -81,9 +81,9 @@ Here are some important cache options correspond to the following settings:
 
 - ``immutable_object_cache_max_size`` The max size for immutable cache.
 
-- ``immutable_object_cache_watermark`` The watermark for the cache. If the
-  capacity reaches to this watermark, the daemon will delete cold cache based
-  on the LRU statistics.
+- ``immutable_object_cache_watermark`` The high water mark for the cache. If the
```

Comment:
s/high water mark/high-water mark/

Reply:
Done, though arguably there should even be a second hypen.